microsoft-edge/webview2/how-to/print.md

```diff
@@ -883,7 +881,7 @@ bool AppWindow::PrintToPdfStream()
                 DisplayPdfDataInPrintDialog(pdfData);
                 std::wstring message = L"Printing " + std::wstring(title.get()) +
                                        L" document to PDF Stream " +
-                                       ((errorCode == S_OK && pdfData != nullptr) ? L"succedded" : L"failed");
+                                       ((errorCode == S_OK && pdfData != nullptr) ? L"succeeded" : L"failed");
```

Comment:
instance 3
@monica-ch - tech review this change in code from succedded to succeeded